judge/pdf_problems.py

```diff
@@ -17,8 +23,10 @@
 PUPPETEER_MODULE = settings.PUPPETEER_MODULE
 HAS_PUPPETEER = os.access(NODE_PATH, os.X_OK) and os.path.isdir(PUPPETEER_MODULE)
 
+HAS_SELENIUM = settings.SELENIUM
```

Comment:
Weird name for `settings.SELENIUM`; doesn't indicate it's a bool.

Reply:
Probably want to use the presence of the actual paths themselves.

Reply:
Selenium will automatically look in path, so they work perfectly with with `None` as their values, hence the use of a boolean at all. Is `USE_SELENIUM` better?